cmd/telemetrygen/internal/logs/config.go

```diff
@@ -28,4 +30,6 @@ func (c *Config) Flags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.Body, "body", "the message", "Body of the log")
 	fs.StringVar(&c.SeverityText, "severity-text", "Info", "Severity text of the log")
 	fs.Int32Var(&c.SeverityNumber, "severity-number", 9, "Severity number of the log, range from 1 to 24 (inclusive)")
+	fs.StringVar(&c.TraceID, "trace-id", "", "TraceID of the log")
+	fs.StringVar(&c.SpanID, "span-id", "", "SpanID of the log")
```

Comment:
If you want to get even fancier, we could change the type of these, move the parsing logic to `UnmarshalText` and use https://pkg.go.dev/flag#TextVar

Reply:
Apparently, this is not available in pflag, which is what we use here :-(